scala/scalafmt/private/format.template.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+RUNPATH="${TEST_SRCDIR-$0.runfiles}"/%workspace%
```

Comment:
Haven't reviewed it or the format-test file since I think a few comments would really make my work easier. Any chance you can document it?

Reply:
Can you write here where do you use it? I have a feeling it's in phase_scalafmt.bzl but couldn't find it

Reply:
In `scala/scalafmt/BUILD`, `format.template.sh` and `format-test.template.sh` are included by `filegroup`. The `filegroup`s are used in extra attributes `_runner` and `_testrunner`. So it is indeed used in `phase_scalafmt.bzl`.

Reply:
👍 wdyt about adding a few comments inside? Skimming it wasn't easy (bash...)

Reply:
I will make it just `RUNPATH`.